config/crds/kr6.0_build_v1alpha1_300-build.yaml

```diff
@@ -0,0 +1,44 @@
+# Copyright 2018 The Knative Authors
```

Comment:
I found the same CRD in `./vendor/github.com/knative/build/config/300-build.yaml`. Maybe we can add a symlink in `config/crd` to it ?

Probably we also have the other CRDs in the `vendor` directory

Reply:
@k15r found out we can also directly reference it from the test suite like [this](https://github.com/ryanbrainard/kubebuilder-p2/blob/4035be7426682c510990779cfb4f4d492921b201/pkg/controller/artist/artist_controller_suite_test.go#L40). Probably better that the symlink

Reply:
Following @k15r  suggestions combined with symlink